technical-reference/reference-technical-articles/xvs-bridge.md

```diff
@@ -4,14 +4,16 @@ This documentation provides detailed instructions and explanations for using the
 * [BNB](https://www.bnbchain.org)
 * [Ethereum](https://ethereum.org)
 * [opBNB](https://opbnb.bnbchain.org)
+* [Arbitrum](https://arbitrum.io)
 
 ## Supported Transfer Paths
 
 The bridge supports transfers between all network pairs, providing users with enhanced flexibility and interoperability across blockchain ecosystems. The supported transfer paths include:
```

Comment:
I would remove the list of pairs, because we are saying that every pair is supported and we previously listed the supported networks.

```suggestion
The bridge supports transfers between all network pairs, providing users with enhanced flexibility and interoperability across blockchain ecosystems.
```

Reply:
Done https://github.com/VenusProtocol/venus-protocol-documentation/pull/204/commits/634f5dace3117ae4201447e8cd91f9d65e5775e6